eventActions/sleepClubActions.js

```diff
@@ -1,6 +1,6 @@
 const config = require('../config.json');
 
-class sleepclubActions {
+class sleepClubActions {
 	static async reactToSleepLog(client, message) {
 		const emote = config.emotes.sleeplogReaction;
```

Comment:
I think this emote was removed, not sure what you want to do about that.

Reply:
Removed the unused emote